cluster/charts/crossplane/values.yaml

```
@@ -22,9 +22,12 @@ tolerations: []
 # -- Add `affinities` to the Crossplane pod deployment.
 affinity: {}
 
-# -- Enable `hostNetwork` for the Crossplane deployment. Caution: enabling `hostNetwork` grants the Crossplane Pod access to the host network namespace.
+# -- Enable `hostNetwork` for the Crossplane deployment. Caution: enabling `hostNetwork` grants the Crossplane Pod access to the host network namespace. Consider setting `dnsPolicy` to `ClusterFirstWithHostNet`.
 hostNetwork: false
 
+# -- Specify the `dnsPolicy` to be used by the Crossplane pod.
+dnsPolicy: "ClusterFirst"
```

Actually, maybe it would be better to leave it empty and wrap it with a conditional later:
```suggestion
dnsPolicy: ""
```

---

Done